src/gsumm.c

```diff
@@ -1037,6 +1061,16 @@ SEXP glast(SEXP x) {
       if (isunsorted) k = oo[k]-1;
       k = (irowslen == -1) ? k : irows[k]-1;
       dans[i] = dx[k];
+      if (narm){
+        j = 0;
+        while(j < grpsize[i]){
+          if (dx[k-j] != NA_REAL){
```

Comment:
this is usually written with `!ISNAN(.)` instead of comparison to `NA_REAL`

Reply:
changed